code/src/WijmoProvider/Features/ColumnReorder.ts

```diff
@@ -14,6 +16,19 @@ namespace WijmoProvider.Feature {
 
         public build(): void {
             this.setState(this._enabled);
+
+            // keep track of group being dragged
+            this._grid.provider.draggingColumn.addHandler((s, e) => {
+                this._draggedColumn = e.getColumn(
+                    true
```

Comment:
Question: What does the true in the getColumn method stand for?

Reply:
![image](https://user-images.githubusercontent.com/3113318/128706949-26627a2e-876d-4241-adf9-85f4aef1ae21.png)